lib/ddtrace/contrib/rails/patcher.rb

```diff
@@ -10,6 +10,7 @@ module Patcher
         option :controller_service
         option :cache_service
         option :database_service
+        option :middleware_names, default: false
```

Comment:
let's keep these functionalities disabled by default; we'll revisit it before going 1.0.